examples/pom.xml

```diff
@@ -297,6 +297,22 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
```

Comment:
Why do you need to add this execution? There's nothing in the "test jar" for the examples module.